libs/hdf-converters/src/ckl-mapper/checklist-mapper.ts

```diff
@@ -296,10 +296,28 @@ function getHdfSpecificDataAttribute(
 }
 
 /**
- * ChecklistResults is a wrapper for ChecklistMapper using the intakeType
- *  default returns a single hdf object without any modifications
- *  split returns multiple hdf object based on number of iSTIG objects in checklist
- *  wrapper returns a single hdf object with an additional profile created using file name as profile name and adds parent_profile key to each mapped profile
+ * The `ChecklistResults` class extends the `ChecklistJsonixConverter` and is responsible for converting
+ * checklist data between different formats (XML CKL, HDF JSON).
+ *
+ * @extends ChecklistJsonixConverter
+ *
+ * @property {string | ExecJSON.Execution} data - The input data, which can be a string of XML data or an HDF JSON execution object.
+ * @property {Checklist} jsonixData - The JSON representation of the checklist data.
```

Comment:
I'd revise this statement to explain better that it's like the jsonix, direct conversion of the ckl, js checklist data in comparison to the intermediate representation which cleans it up and makes it easier to use.